src/presentation/styles/styledComponents/StyledRect.js

```diff
@@ -1,23 +1,28 @@
 import styled from 'styled-components';
-import {chosenPrompt, activeTab} from './StyledHeader.js'
+import {chosenPrompt, activeTab, types} from './StyledHeader.js'
 import '../styleSheets/sass/StyledHeader.scss'
 import React from 'react';
 import { OutterRect, InnerRect, outterTextStyle, innerTextStyle } from './StyledConstants.js';
 
+var CurrentMessage = {[types[0]]: " - Hello, here are some things I can help you with:",[types[1]]: "- Hi, here are some things I can help you with:"}
```

Comment:
I think anything pertaining the active tab or active prompt should be taken care of in styledHeader, but if it's something constant like types it could be used here.